src/assets/locales/en.json

```diff
@@ -240,6 +240,12 @@
   "home.terms_of_use": "Terms of Use",
   "home.made_with_love": "made with 🦆 by ",
   "home.environment_author_by": " by ",
+  "home.rooms-title": "Instantly create rooms",
```

Comment:
I think these should use underscores "_" rather than dashes "-".

Reply:
Oh, nice catch.